### PR TITLE
Ignore files in the migration directory other than .js and .coffee.

### PR DESCRIPTION
**Pull request by ms1111**
As per discussion in #28. This skips vim swapfiles, .map files, etc.

### CODE REVIEW

File: src/mongodb-migrations.coffee [hunk 1]
```diff
@@ -176,6 +176,8 @@ class Migrator
         if err
           return cb err
         files = files
+          .filter (f) ->
+            path.extname(f) in ['.js', '.coffee'] && !f.startsWith('.')
```
Comment (emirotin) [inline src/mongodb-migrations.coffee]: Can you please use `and not ` here? A matter of style, but consistent with the rest of the code.

File: test/migrations/4-test4.coffee [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports.id = "4-test4"
```
Comment (emirotin) [inline test/migrations/4-test4.coffee]: Why is this file needed? Does it demonstrate any new concept or the edge case?
Also the ID style is different, others don't have the order number in the ID.

**Comment by emirotin** [comment]
Thanks for the submission!

File: test/migrations/4-test4.coffee (hunk 2 above)
Comment (ms1111) [inline test/migrations/4-test4.coffee]: I've renamed it to reflect that it's testing the whitelist matches .coffee files.

Comment (emirotin) [inline test/migrations/4-test4.coffee]: Ah, I've missed it's in CS, makes sense

**Comment by emirotin** [comment]
Published as 0.8.4
